app/livechat/server/methods/startFileUploadRoom.js

```diff
@@ -16,6 +16,7 @@ Meteor.methods({
 			token: guest.token,
 		};
 
-		return Livechat.getRoom(guest, message);
+		const room = await Livechat.getRoom(guest, message);
```

Comment:
since you'll just return `room`, you don't need to `await` from `Livechat.getRoom` result, you can return `return Livechat.getRoom()` and remove the `async` from method.